From de44c6098d70eed1af5eebef0aaf20639d30685f Mon Sep 17 00:00:00 2001
From: Ilia Sergachev <isergachev@nvidia.com>
Date: Mon, 10 Mar 2025 15:05:35 -0700
Subject: [PATCH] PR #23509: [GPU] Add cuDNN 9.8.0.

Imported from GitHub PR https://github.com/openxla/xla/pull/23509

Copybara import of the project:

--
913743e282a952ca475678a5c713bd1ac502dd2f by Ilia Sergachev <isergachev@nvidia.com>:

[GPU] Add cuDNN 9.8.0.

Merging this change closes #23509

PiperOrigin-RevId: 735525100
---
 third_party/gpus/cuda/hermetic/cuda_redist_versions.bzl       | 4 ++++
 .../third_party/gpus/cuda/hermetic/cuda_redist_versions.bzl   | 4 ++++
 2 files changed, 8 insertions(+)

diff --git a/third_party/gpus/cuda/hermetic/cuda_redist_versions.bzl b/third_party/gpus/cuda/hermetic/cuda_redist_versions.bzl
index 747c89f2d43a7f..b65866b6cd9688 100644
--- a/third_party/gpus/cuda/hermetic/cuda_redist_versions.bzl
+++ b/third_party/gpus/cuda/hermetic/cuda_redist_versions.bzl
@@ -133,6 +133,10 @@ CUDNN_REDIST_JSON_DICT = {
         "https://developer.download.nvidia.com/compute/cudnn/redist/redistrib_9.7.1.json",
         "f9bc411a4908f0931e7323f89049e3a38453632c4ac5f4aa3220af69ddded9dc",
     ],
+    "9.8.0": [
+        "https://developer.download.nvidia.com/compute/cudnn/redist/redistrib_9.8.0.json",
+        "a1599fa1f8dcb81235157be5de5ab7d3936e75dfc4e1e442d07970afad3c4843",
+    ],
 }
 
 CUDA_12_NCCL_WHEEL_DICT = {
diff --git a/third_party/xla/third_party/gpus/cuda/hermetic/cuda_redist_versions.bzl b/third_party/xla/third_party/gpus/cuda/hermetic/cuda_redist_versions.bzl
index 747c89f2d43a7f..b65866b6cd9688 100644
--- a/third_party/xla/third_party/tsl/third_party/gpus/cuda/hermetic/cuda_redist_versions.bzl
+++ b/third_party/xla/third_party/tsl/third_party/gpus/cuda/hermetic/cuda_redist_versions.bzl
@@ -133,6 +133,10 @@ CUDNN_REDIST_JSON_DICT = {
         "https://developer.download.nvidia.com/compute/cudnn/redist/redistrib_9.7.1.json",
         "f9bc411a4908f0931e7323f89049e3a38453632c4ac5f4aa3220af69ddded9dc",
     ],
+    "9.8.0": [
+        "https://developer.download.nvidia.com/compute/cudnn/redist/redistrib_9.8.0.json",
+        "a1599fa1f8dcb81235157be5de5ab7d3936e75dfc4e1e442d07970afad3c4843",
+    ],
 }
 
 CUDA_12_NCCL_WHEEL_DICT = {
